codegenerator/cli/templates/static/codegen/src/db/Migrations.res

```diff
@@ -180,22 +207,3 @@ let runDownMigrations = async (~shouldExit) => {
   }
   exitCode.contents
 }
-
-let setupDb = async () => {
-  Logging.info("Provisioning Database")
-  // TODO: we should make a hash of the schema file (that gets stored in the DB) and either drop the tables and create new ones or keep this migration.
-  //       for now we always run the down migration.
-  // if (process.env.MIGRATE === "force" || hash_of_schema_file !== hash_of_current_schema)
-  let exitCodeDown = await runDownMigrations(~shouldExit=false)
-  // else
-  //   await clearDb()
-
-  let exitCodeUp = await runUpMigrations(~shouldExit=false)
-
-  let exitCode = switch (exitCodeDown, exitCodeUp) {
-  | (Success, Success) => Success
-  | _ => Failure
-  }
-
-  process->exit(exitCode)
```

Comment:
It looks like we no longer call process.exit with exit code. This could be a problem for the hosted service

Reply:
We call it in `runUpMigrations`, but the first `await sql->unsafe(` actually don't have it. Should I add it back? Won't it exit with 1 automatically if the query fails?

Reply:
All of the other potential failures are caught and only returned in the exit code. We definitely need the exit code to work accurately for the hosted service so I would suggest we keep the same logic.